### PR TITLE
Update github workflows

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

**Comment by Mathiamu** [comment]
Hey, just copy pasting the workflows from portefoljeflate will not work.. Many things are different. Maofs and vedtaksstottefs should have the same files, but we can't blindly use the ones from portefoljeflate

**Comment by dragutin-nav** [comment]
@Mathiamu I've fixed what was easy to spot, so can you please check again.

File: .github/workflows/deploy-dev-gcp.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+                  cache: npm
+            - name: Install dependencies
+              run: npm ci
+            - name: Run tests
+              run: npm run test
+            - name: Build application
+              run: npm run build:gcp
```
Comment (Mathiamu) [inline .github/workflows/deploy-dev-gcp.yaml:28]: We don't have a `build:gcp` command, only `build`

File: .github/workflows/update-github-pages.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Deploy github pages
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - master
```
Comment (Mathiamu) [inline .github/workflows/update-github-pages.yaml:6]: Either remove this or remove the step from deploy-prod, we don't need to do it twice!

File: .github/workflows/deploy-dev-gcp.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: Deploy to dev
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - dev
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply .github/workflows/deploy-dev-gcp.yaml:6]: True

File: .github/workflows/deploy-dev-gcp.yaml (hunk 1 above)
Comment (dragutin-nav) [reply .github/workflows/deploy-dev-gcp.yaml]: Fixed

File: .github/workflows/update-github-pages.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                  cache: 'npm'
+            - name: Install dependencies
+              run: npm ci
+            - name: Build mock application
+              run: npm run build:mock
+            - name: Update GitHub pages
+              uses: peaceiris/actions-gh-pages@v3
+              with:
+                  github_token: ${{ secrets.GITHUB_TOKEN }}
+                  publish_dir: ./build
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply .github/workflows/update-github-pages.yaml]: Fixed

**Pull request merged**